### PR TITLE
Create tests for DAV ID icon -issue #28

### PR DESCRIPTION
**Pull request by Valilutzik**
Could you please review this pull request?

### CODE REVIEW

File: server/lib/davIdIcon.js [hunk 1]
```diff
@@ -1,5 +1,16 @@
 const getDavIdIconUrl = () => 'http://lorempixel.com/100/100/abstract/';
+const { URL } = require('url');
```
Comment (TalAter) [inline server/lib/davIdIcon.js]: There's no need to modify the original file for testing purposes.

Keep all the test related code in the test file.

Comment (Valilutzik) [reply server/lib/davIdIcon.js]: Ok, I have updated the files, will push changes now

**Comment by Valilutzik** [comment]
Thanks for your answer @TalAter, I added it when trying to work on issue #36 and forgot it. when running `npm test` test fails because of it.

**Comment by TalAter** [comment]
Works great now!

Thank you, @Valilutzik. I appreciate all the help. Hopefully, we'll be seeing many more contributions from you!

**Pull request merged**